pipeline.yml

```diff
@@ -108,12 +108,7 @@ jobs:
           UAA_API_URL: ((uaa-address-staging))
           UAA_CLIENT_ID: ((uaa-client-id-test-staging))
           UAA_CLIENT_SECRET: ((uaa-client-secret-test-staging))
-          CLIENT_SERVICE_NAME: cloud-gov-identity-provider
-          USER_SERVICE_NAME: cloud-gov-service-account
-          CLIENT_PLAN_NAME: oauth-client
-          USER_PLAN_NAME: space-deployer
-          SERVICE_INSTANCE_NAME: uaa-credentials-acceptance
-
+         
   - name: push-broker-production
```

Comment:
These were removed because we're just hardcoding the names into the acceptance tests script, since we want to test all possible plans anyway? I'm not disagreeing with it, just asking

Reply:
Those are coded in the broker catalog. They shouldn't be params in my opinion as they are the values we are testing for.